RecoTauTag/RecoTau/src/DeepTauBase.cc

```diff
@@ -26,11 +26,11 @@ namespace deep_tau {
     if (!simple_value) {
       static const std::string prefix =
           "[&](double *x, double *p) { const int decayMode = p[0];"
-          "const double pt = p[1]; const double eta = p[2];";
+          "const double pt = p[1]; const double eta = p[2]; return ";
```

Comment:
Originally `return ` was not added here to keep a possibility of having more complicated cut expressions like, for example, this one https://github.com/cms-sw/cmssw/blob/master/RecoTauTag/RecoTau/python/tools/runTauIdMVA.py#L745-L748

I would prefer to keep such flexibility. But to improve user-friendliness for not so complicated cut expressions, you can modify the code in the following way:
```c++
std::string fn_str = prefix;
if(cut_str.find("return") == std::string::npos)
  fn_str += " return " + cut_str + ";}";
else
  fn_str += cut_str + "}";
```